src/routes/spotify/playlist.svelte.ts

```diff
@@ -120,14 +120,34 @@ export const Playlist = () => {
     }
 
     // parse text into dates
-    var re = /(\d{4})-(\d{2})-(\d{2})T(\d{2}):(\d{2}):(\d{2})/; // startswith: 2015-04-29T22:06:55
+    const re = /(\d{4})-(\d{2})-(\d{2})T(\d{2}):(\d{2}):(\d{2})/; // startswith: 2015-04-29T22:06:55
     albums = JSON.parse(text, (k, v) => {
       if (typeof v == "string" && re.test(v)) {
         return new Date(v);
       }
       return v;
     });
 
+    albums.forEach((a) => {
+      const titleRe = /\s+[([].*(deluxe|remaster|expanded|anniversary).*[)\]]$/i;
```

Comment:
should I pull this stuff to a test already? I see there are some 🤔

Reply:
If it'll help cover these more! But some of the live tests probably won't work for you

Reply:
yeah let me slap a quick one, this is for sure the type of thing we can be tweaking for a while

Reply:
pushed